WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2286,15 +2292,9 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                     }
                     break;
                 case RequestCodes.IMAGE_EDITOR_EDIT_IMAGE:
-                    if (data.hasExtra(UCrop.EXTRA_OUTPUT_URI)) {
```

Comment:
I've consolidated the results from the ImageEditor and decided to always use the same path. The result contains either a list with a single item or a list with multiple items.